unit_tests/test_config.py

```diff
@@ -638,49 +637,13 @@ def test_20_user_data_format_version_azure(self):
         # returns something from 5 months ago.
         self.assertIn('user_data_format_version', conf)
 
-    def test_21_verify_unique_stress_cmds_per_tenant(self):
```

Comment:
I would like to see equivalent unittest for those, instead of just removing them

Reply:
Ok. Will add.

Reply:
Done